src/panels/lovelace/entity-rows/hui-weather-entity-row.ts

```diff
@@ -0,0 +1,301 @@
+            "ui.panel.lovelace.warning.entity_not_found",
+            "entity",
+            this._config.entity
+          )}</hui-warning
+        >
+      `;
+    }
+
+    const pointer =
+      (this._config.tap_action && this._config.tap_action.action !== "none") ||
+      (this._config.entity &&
+        !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(this._config.entity)));
+
+    const secondaryAttribute = this._getSecondaryAttribute(stateObj);
+    const extrema = this._getExtrema(stateObj);
+
+    return html`
+      <div
+        class="main ${classMap({
+          pointer,
+        })}"
+        @action=${this._handleAction}
+        .actionHandler=${actionHandler({
+          hasHold: hasAction(this._config!.hold_action),
+          hasDoubleClick: hasAction(this._config!.double_tap_action),
+        })}
+        tabindex=${ifDefined(pointer ? "0" : undefined)}
+      >
+        <state-badge
+          .hass=${this.hass}
+          .stateObj=${stateObj}
+          .overrideImage=${weatherIcons[stateObj.state]}
+        ></state-badge>
+        <div class="container">
+          <div style="display: flex;">
+            <div class="info">
+              <div class="name">
+                ${this._config.name || computeStateName(stateObj)}
+              </div>
+              <div class="state">
+                ${this.hass.localize(`state.weather.${stateObj.state}`) ||
+                  stateObj.state}
+              </div>
+            </div>
+            <div class="temperature">
+              ${stateObj.attributes.temperature}<span
+                >${getWeatherUnit(this.hass, "temperature")}</span
+              >
+            </div>
+          </div>
+          <div class="attributes">
+            ${secondaryAttribute
+              ? html`
+                  <span>
+                    ${secondaryAttribute}
+                  </span>
+                `
+              : ""}
+            ${secondaryAttribute && extrema
+              ? html`
+                  <span>|</span>
+                `
+              : ""}
+            ${extrema
+              ? html`
+                  <span>
+                    ${extrema}
+                  </span>
+                `
+              : ""}
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  private _getSecondaryAttribute(stateObj: WeatherEntity): string | undefined {
+    if (!stateObj.attributes.forecast?.length) {
+      return undefined;
+    }
+
+    const forecastNow = stateObj.attributes.forecast[0];
+
+    let value: string;
+    let attribute: string;
+
+    if (forecastNow.precipitation) {
+      value = forecastNow.precipitation.toString();
+      attribute = "precipitation";
+    } else if (forecastNow.humidity) {
+      value = forecastNow.humidity.toString();
+      attribute = "humidity";
+    } else {
+      return undefined;
+    }
+
+    if (attribute === "wind_bearing") {
+      const cardinalDirection = getWindBearing(value);
```

Comment:
You get the wind bearing of the humidity/precipitation?

Reply:
hmm left over code that never got removed. SMH